modules/core/task/main.tf

```diff
@@ -30,6 +30,22 @@ resource "aws_ecs_task_definition" "main" {
           labels        = lookup(docker_volume_configuration.value, "labels", null)
         }
       }
+      dynamic "efs_volume_configuration" {
+        for_each = lookup(volume.value, "efs_volume_configuration", null) == null ? [] : [volume.value["efs_volume_configuration"]]
+        content {
+          file_system_id          = lookup(efs_volume_configuration.value, "file_system_id", null)
+          root_directory          = lookup(efs_volume_configuration.value, "root_directory", null)
+          transit_encryption      = lookup(efs_volume_configuration.value, "transit_encryption", "ENABLED")
+          transit_encryption_port = lookup(efs_volume_configuration.value, "transit_encryption_port", null)
+          dynamic "authorization_config" {
+            for_each = lookup(efs_volume_configuration.value, "authorization_config", null) == null ? [] : [efs_volume_configuration.value["authorization_config"]]
+            content {
+              access_point_id       = lookup(authorization_config.value, "access_point_id", null)
+              iam                   = lookup(authorization_config.value, "iam", "DISABLED")
```

Comment:
I don't know my tf from my aws from my pinky, but what is this `DISABLED` constant?
I've never seen it before 🤔

Reply:
Ah, specified here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition oh well I guess I really don't know sh!t :(

Reply:
👍 Just thought I'd put in a sensible default so it's explicit.
Not sure why linting failed on the PR, it doesn't really say anything useful! let me know if I need to fix it up more

Reply:
Please run `terraform fmt`

Lint failed because terraform fmt reformatted the file, thanks

Reply:
I think Michael is on vacation, but will be back soon.